packages/service-library/src/servicelib/fastapi/prometheus_instrumentation.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=protected-access
+
+
+from fastapi import FastAPI
+from prometheus_fastapi_instrumentator import Instrumentator
+
+
+def instrument_app(app: FastAPI):
+
+    instrumentator = (
+        Instrumentator(
```

Comment:
Q: What kind of metrics is this instrumentator collecting? 
- In aiohttp we had a lower level interface and we had to specify sensors etc ...
- Moreover, we had issues with cardinality that would totally overload the system.
- Have you tested the impact of the reading on `/metric` on the system  (NOTE: in the past these handlers were sync and therefore blocking over-time )

Reply:
I think I saw that we in the `aiohttp` use https://pypi.org/project/prometheus-client/. The package I use here builds on top of that to have a simple interface for fastapi. 
Concerning which metrics are collected, you can see that [here](https://github.com/trallnag/prometheus-fastapi-instrumentator#prometheus-fastapi-instrumentator-). In addition to the default ones I collect the number of requests currently in progress, which I believe is very useful. 
I have not tested the performance impact, but the developer makes [this](https://github.com/trallnag/prometheus-fastapi-instrumentator#disclaimer) disclaimer about the package. However, my point of view is that we need to collect some metrics and I would prefer to use a tool rather than define the lowlevel metrics myself. So until I see this one causes issues I think we should go for it.